catalyst-gateway/event-db/Earthfile

```diff
@@ -9,7 +9,7 @@ VERSION 0.7
 # Internal: builder is our Event db builder target. Prepares all necessary artifacts.
 #   CI target : dependency
 builder:
-    DO github.com/input-output-hk/catalyst-ci/earthly/postgresql:v2.11.0+BUILDER \
+    DO github.com/input-output-hk/catalyst-ci/earthly/postgresql:v2.11.1+BUILDER \
```

Comment:
You should be able to run sqlfluff over the entire code base,  it doesn't need to be constrained to `event-db`
sqlfluff is a fast test, you should be able to define it as a check target in the main earthfile.